app-emulation/fuse/fuse-1.5.2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2017 Gentoo Foundation
```

Comment:
Wrong copyright year, repoman would fix that for you and I strongly recommend to use it for basic QA.